client/src/Components/CusButton.jsx

```diff
@@ -1,31 +1,62 @@
 import React, { Component } from "react";
 
 class CusButton extends Component {
+  state = {
+    value: "Type here",
+  };
   dragStart = (e) => {
     const target = e.target;
+    target.classList.add("drag");
+    console.log(target);
     e.dataTransfer.setData("card_id", target.id);
     setTimeout(() => {
       target.style.display = "none";
     }, 0);
   };
+  handleInputChange = (e) => {
+    const currentTarget = e.currentTarget;
+    const value = currentTarget.value;
+    this.setState({ value });
+  };
   dragover = (e) => {
     e.stopPropagation();
   };
+  getInputStyles = (color, radius) => {
+    console.log("get Input styles" + color);
+    const styles = {};
+    styles.color = `white`;
+    styles.display = "block";
+    styles.borderRadius = `${radius}`;
+    styles.backgroundColor = `${color}`;
```

Comment:
you are not reusing the object styles again so avoid creating a const and simply 
return {
objectData
}